opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
-  //
-  // Note: only [a, b) intervals are currently supported for each bucket. If we decides
-  // to also support (a, b] intervals we should add support for these by defining a boolean
-  // value which decides what type of intervals to use.
-  repeated double explicit_bounds = 7;
+  // The set of labels associated with the metric descriptor. Labels in this
+  // list apply to all data points.
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 7;
 }
 
-// SummaryDataPoint is a single data point in a timeseries that describes the time-varying
-// values of a Summary metric.
-message SummaryDataPoint {
-  // The set of labels that uniquely identify this timeseries.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
-
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
-  //
-  // The cumulative value is over the time interval (start_time_unix_nano, time_unix_nano].
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the timestamp
-  // may be decided by the backend.
-  fixed64 start_time_unix_nano = 2;
-
-  // time_unix_nano is the moment when this value was recorded.
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  fixed64 time_unix_nano = 3;
-
-  // The total number of recorded values since start_time. Optional since
-  // some systems don't expose this.
-  uint64 count = 4;
-
-  // The total sum of recorded values since start_time. Optional since some
-  // systems don't expose this. If count is zero then this field must be zero.
-  double sum = 5;
-
-  // Represents the value at a given percentile of a distribution.
-  //
-  // To record Min and Max values following conventions are used:
-  // - The 100th percentile is equivalent to the maximum value observed.
-  // - The 0th percentile is equivalent to the minimum value observed.
-  //
-  // See the following issue for more context:
-  // https://github.com/open-telemetry/opentelemetry-proto/issues/125
-  message ValueAtPercentile {
+// Distribution is a data point in a timeseries containing statistics for
+// an observed population of values.
+message Distribution {
+  // count is the number of values in a population.
+  uint64 count = 1;
+
+  // sum is the total sum of all values in a population.
+  double sum = 2;
+
+  double minimum = 3;
+  double maximum = 4;
+
+  repeated uint64 bucket_counts = 5;
+
+  message BucketBounds {
+    oneof BoundKinds {
+      ExplicitBounds explicit_bounds = 1;
+      LinearBounds linear_bounds = 2;
+    }
+  }
+  BucketBounds bucket_bounds = 6;
+
+  message Percentile {
     // The percentile of a distribution. Must be in the interval
     // [0.0, 100.0].
     double percentile = 1;
 
     // The value at the given percentile of a distribution.
     double value = 2;
   }
+  repeated Percentile percentiles = 7;
```

Comment:
Should this really be a first-class citizen? Since percentiles cannot be aggregated, supporting it directly sounds like encouraging bad practices.

Reply:
I agree. It is influenced by both the previous proto definition that had a `Summary` type to support this, and both the Prometheus' `Summary` and part of Stackdriver's `Distribution`.

I whole hardily agree these lead users to make nonsense aggregations, but was keeping it around to answer the opposite question of why did you remove them.

If there is a consensus for not including these I'm 100% onboard to dropping.

Reply:
To add to this, in my experience, I have seem limited use of Prometheus `Summary`s for this exact reason. It is much more flexible to send the distribution data as a histogram and calculate a close approximation of an arbitrary percentile when needed (not when measured).

I think if you wanted to optimization of sending percentiles, you could send them as Int64 or Double data on your own.